### ISSUE TITLE
golem-cli first-time initialization is confusing to users

### ISSUE THREAD

**Issue by vigoo** [golemcloud/golem#647]
See https://discord.com/channels/1134448700572319785/1176446164393013269/1259237497687380148

Not sure what the best solution is, but some ideas:

- list all commands in `--help` but mark the ones that need init somehow
- do not list any commands in `--help` and just display more detailed information about the need to init
- make it more clear how to init for the default golem-oss docker (have a simple to find default option, don't ask hosts/ports the user does not know)
- Use default docker compose for OSS CLI profile configuration

**Comment by afsalthaj** [comment]
Managing the profile after the first initialisation can be confusing too. For example: Let' say I created an OSS profile and I was using it and then later down the line, I decided to create a cloud profile and switch to it. I have to do that using

```scala
golem-cli profile add golem-cloud something
Unexpected client error: error sending request for url (http://localhost:8083/version): error trying to connect: tcp connect error: Connection refused (os error 61)
Profile created
```

It's good if it doesn't try to connect to the servers in the OSS profile, if all we need is a new cloud profile. It also says profile is created. I think important to solve this before 1.0.
